check/pytest2

```diff
@@ -35,7 +35,7 @@ trap "exit" INT
 # Convert code from python3 to python2.7.
 echo "Transpiling to python 2 with destination '${out_dir}'..."
 set -e
-./python2.7-generate.sh "${out_dir}"
+./dev_tools/python2.7-generate.sh "${out_dir}"
 set +e
 
```

Comment:
`.readthedocs.yml` doesn't belong in `continuous-integration`. It's not really related to that concept.